### PR TITLE
Add loading skeleton for price calculator

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Added loading indicator for PriceCalculator.  See attached video (loading speed artifically slowed down)

<div class='graphite__hidden'>
          <div>🎥 Video uploaded on Graphite:</div>
            <a href="https://app.graphite.dev/media/video/GLY3tSM85RFBvAf6lFNA/a3107499-4988-4e60-b340-874725ba6250.mov">
              
            </a>
          </div>
<video src="https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/GLY3tSM85RFBvAf6lFNA/a3107499-4988-4e60-b340-874725ba6250.mov">Screen Recording 2024-02-29 at 15.01.38.mov</video>

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3999?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* **#3999** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3999?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#3998** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3998?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3997** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3997?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3991** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3991?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3985** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3985?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3984** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3984?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#3982** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3982?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3999?utm_source=stack-comment).

File: apps/store/src/components/PriceCalculator/PriceCalculatorDynamic.css.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+export const sectionSkeleton = style({
+  height: theme.space.xxl,
+  width: '100%',
+  selectors: {
+    '&:first-child': {
+      marginTop: theme.space.md,
+    },
+  },
+})
```
Comment (gustaveen) [inline apps/store/src/components/PriceCalculator/PriceCalculatorDynamic.css.ts:12]: Would be nice to match the height of the skeleton to the inputs/button/sections of the form :) 

Screenshot 2024-02-29 at 15 26 51

**Review by gustaveen — APPROVED** [review]
☠️

**Review by guilhermespopolin — APPROVED** [review]
Looking polished :)
Gustav suggestion of marking the skeleton the same size as the inputs would take it to the next level for sure.

File: apps/store/src/components/PriceCalculator/PriceCalculatorDynamic.css.ts (hunk 1 above)
Comment (alebedev) [reply apps/store/src/components/PriceCalculator/PriceCalculatorDynamic.css.ts:12]: We have 4 elements of 3 sizes.  Ended up matching those manually

**Comment by alebedev** [comment]
### Merge activity

* **Mar 1, 6:02 AM EST**: @alebedev merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3999).

**Pull request merged**